packages/eslint-config-loris/package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "peerDependencies": {
     "eslint": "^3.2.0",
-    "eslint-plugin-react": "^7.0.0"
+    "eslint-plugin-react": "^7.1.0"
```

Comment:
Can you revert this changes? We up version in separate commit with changelog modification, e.g. https://github.com/ymaps/codestyle/commit/67b04a410208761c13372237864a89cfaf76f37b